packages/sdk/src/api/prover/prover.ts

```diff
@@ -45,7 +45,7 @@ export async function prove<T extends Abi, F extends ContractFunctionName<T>>(
   args: ContractFunctionArgs<T, AbiStateMutability, F>,
   chainId: number = foundry.id,
   url: string = "http://127.0.0.1:3000",
-  gasLimit: number = 10_000_000,
+  gasLimit: number = 400,
```

Comment:
Make it a rounder number, e.g. 500:

```suggestion
  vgasLimit: number = 500,
```

Reply:
Will rename it to `vgasLimit` in the next PR.